api/rosetta/data_transaction_integration_test.go

```diff
@@ -0,0 +1,554 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// +build integration
+
+package rosetta_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/labstack/echo/v4"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/model/flow"
+
+	"github.com/optakt/flow-dps/api/rosetta"
+	"github.com/optakt/flow-dps/models/dps"
+	"github.com/optakt/flow-dps/rosetta/configuration"
+	"github.com/optakt/flow-dps/rosetta/identifier"
+	"github.com/optakt/flow-dps/rosetta/meta"
+	"github.com/optakt/flow-dps/rosetta/object"
+)
+
+func TestGetTransaction(t *testing.T) {
```

Comment:
```suggestion
func TestAPI_Transaction(t *testing.T) {
```

Reply:
To be honest I find the underscored version a bit less readable :/ Though I do find it mildly upsetting to have the names without a clear pattern:
- `TestGetTransaction`
- `TestTransactionErrors`
- `TestMalformedTransactionRequest`

Reply:
I didn't like it much either, but it's in line with the official Go recommendations and used by some popular tooling, see:

https://golang.org/pkg/testing/#hdr-Examples

So I would suggest we follow that, as was agreed upon in the discussion related to testing:

https://github.com/optakt/flow-dps/discussions/100

Reply:
Once you're used to it, it adds a nice layer of readability to the output of `go test -v ./...` since you can directly see which structs are being tested as opposed to just random names of tests that might repeat themselves and not make much sense out of context.